pkg/agent/metrics/metrics.go

```diff
@@ -35,17 +37,31 @@ const (
 )
 
 var (
+	// Use buckets ranging from 5 ms to 12.5 seconds.
+	latencyBuckets = []float64{0.005, 0.025, 0.1, 0.5, 2.5, 12.5}
```

Comment:
I know we've had problems on the KAS with the top end bucket onlye being 8 seconds. A little worried that 12 seconds is still a little too short for the top bucket. What about something like {0.005, 0.025, 0.1, 0.5, 2.5, 10, 30}?

Reply:
Done.